Src/AutoFixture.NUnit3/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("AutoFixture.NUnit3")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Ploeh")]
+[assembly: AssemblyProduct("AutoFixture.NUnit3")]
+[assembly: AssemblyCopyright("Copyright © Ploeh 2011")]
```

Comment:
Not enough I guess I'll update it. Missed it when copying from AutoFixture.NUnit2...